src/cascadia/TerminalApp/TerminalPage.cpp

```diff
@@ -4434,6 +4434,19 @@ namespace winrt::TerminalApp::implementation
 
         til::color bgColor = backgroundSolidBrush.Color();
 
+        const auto getTerminalBrush = [this]() -> Media::Brush {
+            if (const auto& control{ _GetActiveControl() })
+            {
+                return control.BackgroundBrush();
+            }
+            else if (auto settingsTab = _GetFocusedTab().try_as<TerminalApp::SettingsTab>())
+            {
+                return settingsTab.Content().try_as<Settings::Editor::MainPage>().BackgroundBrush();
+            }
+            return nullptr;
+        };
+        const auto terminalBrush = getTerminalBrush();
```

Comment:
I was about to comment asking why this lambda is needed, when you can also just do this:
```cpp
Brush terminalBrush{ nullptr };
if (const auto& control{ _GetActiveControl() })
{
    terminalBrush = control.BackgroundBrush();
}
else if (auto settingsTab = _GetFocusedTab().try_as<TerminalApp::SettingsTab>())
{
    terminalBrush = settingsTab.Content().try_as<Settings::Editor::MainPage>().BackgroundBrush();
}
```

but then I realized the code only moved. Still, you could consider making this change since it got changed anyways.